packages/mds-ingest-service/tests/index.spec.ts

```diff
@@ -191,6 +191,30 @@ const TEST_EVENT_ANNOTATION_B: EventAnnotationDomainCreateModel = {
   latency_ms: 150
 }
 
+const TEST_EVENT_ANNOTATION_C = {
+  events_row_id: 1,
+  device_id: DEVICE_UUID_A,
+  timestamp: testTimestamp,
+  vehicle_id: 'test-id-1',
+  vehicle_type: 'scooter' as VEHICLE_TYPE,
+  propulsion_types: ['electric'] as PROPULSION_TYPE[],
+  geography_ids: [uuid(), uuid()],
+  geography_types: ['jurisdiction', null],
+  latency_ms: 100
+}
+
+const TEST_EVENT_ANNOTATION_D = {
+  events_row_id: 2,
+  device_id: DEVICE_UUID_B,
+  timestamp: testTimestamp,
+  vehicle_id: 'test-id-2',
+  vehicle_type: 'scooter' as VEHICLE_TYPE,
+  propulsion_types: ['electric'] as PROPULSION_TYPE[],
+  geography_ids: [uuid(), uuid()],
+  geography_types: [null, 'spot'],
+  latency_ms: 150
+}
+
```

Comment:
You should be able to just do
```suggestion
const TEST_EVENT_ANNOTATION_C = {
  ...TEST_EVENT_ANNOTATION_A,
  events_row_id: 1
}

const TEST_EVENT_ANNOTATION_D = {
  ...TEST_EVENT_ANNOTATION_B,
  events_row_id: 2
}
```